apps/system/index.html

```diff
@@ -7,9 +7,14 @@
     <link rel="stylesheet" type="text/css" href="style/system/initlogo.css">
 
     <!-- Shared code -->
+    <!-- these files are loaded lazily but are listed here so they get packaged
+    <script src="shared/js/gzip/gzip_worker.js"></script>
+    <script src="shared/js/gzip/gzip.min.js"></script>
+    -->
     <script src="shared/js/l10n.js"></script>
     <script src="shared/js/l10n_date.js"></script>
     <script src="shared/js/dump.js"></script>
+    <script src="shared/js/gzip/gzip_helper.js"></script>
```

Comment:
Nit: I'd just call this gzip.js.  From an API standpoint, no one needs to know that the gzipping isn't actually being done in this file.